src/utils/init.d.ts

```diff
@@ -0,0 +1,31 @@
+export function init(
+  apiKey: string,
+  options?: {
+    userKey?: null | string;
+    formKeys?: string[];
+    tracking?: 'cookie' | 'fingerprint';
+    authId?: string;
+    authEmail?: string;
+    authPhoneNumber?: string;
+  }
+  // not totally sure about this return type - <void> is the current implicit type
+): Promise<void>;
+
+export function updateUserKey(newUserKey: string, merge?: boolean): void;
+export function setValues(userVals: FieldValues, rerender?: boolean): void;
+export function validateStep(formKey: string, trigger?: boolean): void;
+export function setAuthClient(authClient: any): void;
+export function getAuthClient(): any;
+
+export type FieldValues = {
+  [fieldName: string]:
```

Comment:
This may be slightly pedantic, but the docs use the term 'fieldKey' instead of 'fieldName'.  We should probably keep in consistent.